frontend/coprs_frontend/coprs/models.py

```diff
@@ -1921,6 +1941,14 @@ def isolation_setup(self):
             return {}
         return settings
 
+    def build_done(self):
+        """
+        Record that a build has been submitted into this CoprChroot.
+        """
```

Comment:
Nit: This is an unfortunate name. It reads more like a build just finished. Can we rename to `build_submitted` please?